src/mame/drivers/dreamwld.cpp

```diff
@@ -159,6 +161,7 @@ class dreamwld_state : public driver_device
 	DECLARE_WRITE_LINE_MEMBER(screen_vblank_dreamwld);
 	void draw_sprites( bitmap_ind16 &bitmap, const rectangle &cliprect );
 	required_device<cpu_device> m_maincpu;
+	required_device<i80c52_device> m_mcu;
```

Comment:
What's the point of adding this device finder that's never used?

Reply:
It's for future use when MCU ROM has dumped.

Reply:
Yeah, but there's no point adding a completely unused member, and you don't know if in the future everything will be connected to the MCU via machine configuration, in which case the member still won't be needed.